packages/lesswrong/components/posts/PostsPage/PostsPageSplashHeader.tsx

```diff
@@ -0,0 +1,572 @@
+import { useCurrentUser } from '../../common/withUser';
+import { Coordinates } from './ImageCropPreview';
+import { userIsAdminOrMod } from '../../../lib/vulcan-users';
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    zIndex: theme.zIndexes.postsPageSplashHeader,
+    height: '100vh',
+    marginTop: 'calc(-50px - 64px)',
+    paddingTop: 64,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center top',
+    position: 'relative',
+    textAlign: 'center',
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    ...theme.typography.postStyle,
+    '&::before': {
+      content: '""',
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      height: '100%',
+      width: '100%',
+      pointerEvents: 'none'
+    },
+    transition: 'opacity 0.5s ease-in-out',
+    opacity: 1,
+    [theme.breakpoints.down('sm')]: {
+      marginTop: '-64px',
+      // Cancels out padding for mobile views
+      marginLeft: -8,
+      marginRight: -8
+    },
+  },
+  backgroundImageWrapper: {
+    zIndex: -1, // theme.zIndexes.postsPageSplashHeader,
+    position: 'absolute',
+    paddingTop: 0,
+    marginTop: 'calc(-64px)', // to cancel out the padding in the root class
+    textAlign: 'center',
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    ...theme.typography.postStyle,
+    '&::before': {
+      content: '""',
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      height: '100%',
+      width: '100%',
+      background: `linear-gradient(0deg, ${theme.palette.panelBackground.default} 3%, transparent 48%)`,
+      pointerEvents: 'none'
+    },
+    transition: 'opacity 0.5s ease-in-out',
+    opacity: 1,
+    // The images naturally have a landscape aspect ratio, so when viewing it in a portrait orientation we set height to 100% to scale it appropriately
+    '@media (orientation:portrait)': {
+      height: '100%',
+    },
+    // Also, the images have a width of 2752px, so if the screen is wider(!!!) than that, make sure it covers everything
+    '@media (min-width: 2752px)': {
+      width: '100%',
+    },
+  },
+  backgroundImage: {
+    width: '100%',
+    height: '100%',
+    position: 'relative',
+    zIndex: -2,
+    objectFit: 'cover',
+    objectPosition: 'center top',
+  },
+  backgroundImageCropPreview: {
+    width: '100%',
+    height: '100%',
+    position: 'relative',
+    zIndex: -2,
+    objectFit: 'cover',
+    objectPosition: 'center top',
+  },
+  cropPreviewEnabledForeground: {
+    zIndex: -1,
+  },
+  cropPreviewEnabledBackground: {
+    zIndex: 2,
+  },
+  fadeOut: {
+    '& .PostsPageSplashHeader-backgroundImageWrapper': {
+      opacity: 0,
+      transition: 'opacity 0.5s',
+      transitionTimingFunction: 'ease-in',
+    },
+  },
+  centralSection: {
+    textAlign: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    position: 'relative',
+    color: 'white',
```

Comment:
Can't use absolute colors, need to use `theme.palette...`.  Not sure what this style in particular is doing and whether it should be responsive to dark mode or not.

Reply:
(will only leave this comment once per file, but using absolute color values like this fails unit tests so all instances need to be fixed.)

Reply:
Fixed for this file